projects/api-client/src/api/build.service.ts

```diff
@@ -140,24 +151,108 @@ export class BuildService {
-    public buildBuildidLogPost(buildid: number, data: MainBuildLog, observe?: 'response', reportProgress?: boolean): Observable<HttpResponse<any>>;
-    public buildBuildidLogPost(buildid: number, data: MainBuildLog, observe?: 'events', reportProgress?: boolean): Observable<HttpEvent<any>>;
-    public buildBuildidLogPost(buildid: number, data: MainBuildLog, observe: any = 'body', reportProgress: boolean = false ): Observable<any> {
+    public getBuildList(limit?: number, offset?: number, orderby?: Array<string>, projectId?: number, scheduledAfter?: Date, scheduledBefore?: Date, finishedAfter?: Date, finishedBefore?: Date, environment?: string, gitBranch?: string, stage?: string, isInvalid?: boolean, status?: 'Scheduling' | 'Running' | 'Completed' | 'Failed', statusId?: number, environmentMatch?: string, gitBranchMatch?: string, stageMatch?: string, match?: string, observe?: 'body', reportProgress?: boolean): Observable<ResponsePaginatedBuilds>;
+    public getBuildList(limit?: number, offset?: number, orderby?: Array<string>, projectId?: number, scheduledAfter?: Date, scheduledBefore?: Date, finishedAfter?: Date, finishedBefore?: Date, environment?: string, gitBranch?: string, stage?: string, isInvalid?: boolean, status?: 'Scheduling' | 'Running' | 'Completed' | 'Failed', statusId?: number, environmentMatch?: string, gitBranchMatch?: string, stageMatch?: string, match?: string, observe?: 'response', reportProgress?: boolean): Observable<HttpResponse<ResponsePaginatedBuilds>>;
+    public getBuildList(limit?: number, offset?: number, orderby?: Array<string>, projectId?: number, scheduledAfter?: Date, scheduledBefore?: Date, finishedAfter?: Date, finishedBefore?: Date, environment?: string, gitBranch?: string, stage?: string, isInvalid?: boolean, status?: 'Scheduling' | 'Running' | 'Completed' | 'Failed', statusId?: number, environmentMatch?: string, gitBranchMatch?: string, stageMatch?: string, match?: string, observe?: 'events', reportProgress?: boolean): Observable<HttpEvent<ResponsePaginatedBuilds>>;
+    public getBuildList(limit?: number, offset?: number, orderby?: Array<string>, projectId?: number, scheduledAfter?: Date, scheduledBefore?: Date, finishedAfter?: Date, finishedBefore?: Date, environment?: string, gitBranch?: string, stage?: string, isInvalid?: boolean, status?: 'Scheduling' | 'Running' | 'Completed' | 'Failed', statusId?: number, environmentMatch?: string, gitBranchMatch?: string, stageMatch?: string, match?: string, observe: any = 'body', reportProgress: boolean = false ): Observable<any> {
 
-        if (buildid === null || buildid === undefined) {
-            throw new Error('Required parameter buildid was null or undefined when calling buildBuildidLogPost.');
-        }
 
-        if (data === null || data === undefined) {
-            throw new Error('Required parameter data was null or undefined when calling buildBuildidLogPost.');
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Cool white space here- is it generated? Anyone know why?

Reply:
/shrug